Bottom: 32bfaf7dd3ec1ab6f66f8d7a51e8e571b5db2027
Top:    b15f15f64074279c567f51d2ad756d7229f5c010
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-20 19:44:18 +0800

Refresh of tap-export-skb_array-0

---

diff --git a/include/linux/if_tap.h b/include/linux/if_tap.h
index d38a1ea..4837157 100644
--- a/include/linux/if_tap.h
+++ b/include/linux/if_tap.h
@@ -3,7 +3,7 @@
 
 #if IS_ENABLED(CONFIG_TAP)
 struct socket *tap_get_socket(struct file *);
-struct skb_array *tap_get_skb_array(struct file *);
+struct skb_array *tap_get_skb_array(struct file *file);
 #else
 #include <linux/err.h>
 #include <linux/errno.h>
